### PR TITLE
Add a contribution page

### PR DESCRIPTION
**Pull request by s-blu**

### CODE REVIEW

File: 00 Meta/Contribution.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# How to contribute
+
+First of all, thanks for considering sharing your knowledge! For adding a query to the vault, there are two things to do: First, **add the query** with fitting test data so it produces results and secondly, **share the query** so it'll become part of the example vault.
+
+## Get a local copy
+
+First of all, you need to get yourself a local copy of the example vault. There are two ways of doing this.
+
+1. Get a static local copy. Follow the [[README#Usage]] explanation for that
```
Comment (dbarenholz) [inline 00 Meta/Contribution.md:9]: Missing period at end of line ;)

Comment (dbarenholz) [reply 00 Meta/Contribution.md:9]: Also: perhaps it might not be clear to others what "static" means.

File: 00 Meta/Contribution.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+2. Get a copy via Github - this is more technical, but makes it possible to update the vault to the newest state automatically and makes sharing your contribution easier.
+
+### Get the example vault via GitHub
+
+You'll need a GitHub Account to follow these steps. If you're not git-savvy, it's probably easier for you to use the [GitHub Desktop](https://desktop.github.com/) application for the following steps.
+
+1. [Create a fork and clone it](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop#forking-a-repository) of the [example vault](https://github.com/s-blu/obsidian_dataview_example_vault) . Now you should have a local copy you can work with.
```
Comment (dbarenholz) [inline 00 Meta/Contribution.md:16]: Extra space after the example vault link ;)

File: 00 Meta/Contribution.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+3. Make your changes like described below!
```
Comment (dbarenholz) [inline 00 Meta/Contribution.md:17]: Inconsistent numbering => "2."

File: 30 Dataview Knowledge/trouble shooting queries.md [hunk 4]
```diff
@@ -4,15 +4,17 @@ Your query is not behaving like it should be? We have some tips on how to find o
 
 ## Query returns 0 results, but the data is there
```
Comment (dbarenholz) [inline 30 Dataview Knowledge/trouble shooting queries.md:5]: Missing period

File: 30 Dataview Knowledge/trouble shooting queries.md [hunk 5]
```diff
@@ -4,15 +4,17 @@ Your query is not behaving like it should be? We have some tips on how to find o
...
 
-There needs to be something in your query thats filtering out or hiding your data.
+There needs to be something in your query that filters out or hiding your data.
```
Comment (dbarenholz) [inline 30 Dataview Knowledge/trouble shooting queries.md:7]: "that filters out or **hides** your data."

File: 30 Dataview Knowledge/trouble shooting queries.md [hunk 6]
```diff
@@ -4,15 +4,17 @@ Your query is not behaving like it should be? We have some tips on how to find o
...
 
 ### 1. Try to remove the FROM clause.
-Does the result you expect show up now (probably among others, not wanted)? Add every WHERE expression back one by one.
+First, remove your `FROM` statement altogether. Does the result you expect show up now (probably among others, not wanted)? Add every FROM expression back one by one.
 
 For example, you have `FROM #dv AND "20 Dataview Queries"`, first add only `FROM #dv`. Your result is still there? Try `FROM #dv AND "20 Dataview Queries"` - do your expected pages vanish? Then the FROM clause - maybe in **combination** with the WHERE clause - is causing trouble. So next...
 
 ### 2. Try to remove the WHERE clause.
-Does the result you expect show up now (probably among others, not wanted)? Add every WHERE expression back one by one.
+Secondly, remove your WHERE clause but keep your `FROM` statement. Are the results still gone? Then your FROM statement is erroneous. Try to find the erroneous part and have a look at your data why they might get filtered out by it.
```
Comment (dbarenholz) [inline 30 Dataview Knowledge/trouble shooting queries.md:15]: `WHERE`

File: 30 Dataview Knowledge/trouble shooting queries.md [hunk 7]
```diff
@@ -4,15 +4,17 @@ Your query is not behaving like it should be? We have some tips on how to find o
...
+
+Your data is there, if you keep the FROM, but delete the WHERE? Then your WHERE statement is acting up.  Add every WHERE expression back one by one.
```
Comment (dbarenholz) [inline 30 Dataview Knowledge/trouble shooting queries.md:17]: Same here: inconsistent codeblocks around FROM and WHERE

File: 30 Dataview Knowledge/trouble shooting queries.md [hunk 8]
```diff
@@ -75,7 +77,7 @@ WHERE paid
 FLATTEN sum() as sum
 ```
 
-It tells us that our **where** clause failed and shows us the **first three errors** why. Here, it's the same one. 
+It tells us that our **where** clause failed and shows us the **first three errors** why. Here, it's the same one three times.
```
Comment (dbarenholz) [inline 30 Dataview Knowledge/trouble shooting queries.md:80]: Replace with `WHERE`

**Review by dbarenholz — COMMENTED** [review]
Decided to hop onto laptop and give a proper review, in stead of a single comment!

File: 00 Meta/Contribution.md (hunk 1 above)
Comment (s-blu) [reply 00 Meta/Contribution.md:9]: Mhh, good point. I'll just throw it out for now.

**Review by dbarenholz — APPROVED** [review]
Wooh, nice.

**Pull request merged**